### PR TITLE
added-option-alice

### PR DESCRIPTION
**Pull request by s21malaz**

### CODE REVIEW

**Comment by s21malaz** [comment]
Please review this, @casey-collab

File: english/bird-salvation/bird.md [hunk 1]
```diff
@@ -4,3 +4,7 @@ marshmallow right off of you.
 	Amazed, you sit in silence at the mythical animal you just witnessed ...
 as well as trying to catch your breath...and, in the distance, glimpse
 what appears to be a horse with a horn sprouting from its forehead...
+
+You really have only one option
+[Go ask Alice](..ask-to-alice/ask-to-alice.md)
```
Comment (casey-collab) [inline english/bird-salvation/bird.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by s21malaz** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :crown:

**Pull request merged**